test/integration/linux/inspec/services_disabled_spec.rb

```diff
@@ -30,8 +30,11 @@
   it { should_not be_enabled }
 end
 
-describe service('whoopsie') do
-  it { should_not be_enabled }
+# This check is broken on Ubuntu 22.04, even when the service is disabled.
```

Comment:
Is whoopsie even present/used in 2204?

Reply:
Yes, but from what I can see it is a static service so you cannot disable it.

Reply:
@aerickson I manually ran `systemctl disable whoopsie` and it still fails on the test, so I think the test is broken or something changed that it works differently. On the raw `ubuntu:jammy` image whoopsie does not exist. I did see a `apt install` line for whoopsie in the output logs.. But I cannot find anything referencing it in the code.

Reply:
Setting it to disabled will make Puppet install it. Not intuitive, but it can't set disabled if the package isn't present.

Let's just do nothing regarding whoopsie on 2204 (ie in the 2204 block do nothing).